gax/add-cjs-extension.cjs

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
```

Comment:
Why not add a package.json with `type: commonjs`? According to spec:
- https://nodejs.org/api/packages.html#packagejson-and-file-extensions

Additionally reading/examples for this pattern:
- https://medium.com/ekino-france/supporting-dual-package-for-cjs-and-esm-in-typescript-library-b5feabac1357
- https://github.com/azu/tsconfig-to-dual-package
- https://github.com/knightedcodemonkey/duel

Reply:
This is for changing the extension in proxyquire calls, so it wouldn't be affected by a package.json, although I see your larger point that we wouldn't even have to do that if we didn't rename the files at all. It seems slightly cleaner to me to have the .cjs and .js distinguishing the files, easier to see at a glance what's going on, but that's just my perspective!